docs/dev-process/pull-request-process.md

```diff
@@ -0,0 +1,51 @@
+# Pull Request Process
```

Comment:
There is a duplicate section of the same content in https://github.com/PowerShell/PowerShell/blob/master/docs/maintainers/README.md#pull-request-workflow
We should remove it from the README.md.

Reply:
I'm going to move this content out of README.md and get rid of this doc and move everything into CONTRIBUTING.md